src/dotty/tools/dotc/transform/ExtensionMethods.scala

```diff
@@ -88,11 +88,10 @@ class ExtensionMethods extends MiniPhaseTransform with DenotTransformer with Ful
             decls1.enter(evt2uSym)
           }
 
-          // add a VCXXXCompanion superclass
```

Comment:
I'd better leave this comment as is, it explains what should be happening in next lines.